packages/camera/lib/camera_image.dart

```diff
@@ -84,8 +92,10 @@ class ImageFormat {
 ImageFormatGroup _asImageFormatGroup(dynamic rawFormat) {
   if (defaultTargetPlatform == TargetPlatform.android) {
     // android.graphics.ImageFormat.YUV_420_888
-    if (rawFormat == 35) {
-      return ImageFormatGroup.yuv420;
+    if (rawFormat == 256) {
+      return ImageFormatGroup.jpeg;
+    } else if (rawFormat == 42) {
+      return ImageFormatGroup.bgra8888;
```

Comment:
This contradicts [Camera.java:91](https://github.com/flutter/plugins/blob/cd7b678ff548c93fe377c5d16dede2210a959917/packages/camera/android/src/main/java/io/flutter/plugins/camera/Camera.java#L91): there, you don't allow **bgra8888** format for Android, but only **jpeg** or **yuv420**.

Reply:
yeah, you are right, on first iteration, I thought about supporting bgra on both android and ios since its already on ios side, but then realized its better to feed jpeg, I missed this on the second iteration it seems. pushed new fix